mne/coreg.py

```diff
@@ -77,9 +77,8 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
 
     Parameters
     ----------
-    mne_root : None | str
-        The mne root directory (only needed if MNE_ROOT is not specified as
-        environment variable).
+    mne_root : None
+        This argument is not used anymore.
```

Comment:
Hmm... I guess you can leave it if you want then.

However, I wouldn't hold my breath about making `fsaverage` downloadable. I don't think the FS license permits doing that, but you're welcome to ask them and see.